src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameService.java

```diff
@@ -370,11 +391,13 @@ private Map<Integer, CardState> mapCardsState(CardCollection cardCollection) {
     private Game handleMatch(Game currentGame, Turn currentTurn) {
         if (checkMatch(currentGame, currentTurn)) {
             if (isCompleteSet(currentGame, currentTurn)) {
+                pausePlaybackAllPlayers(currentGame);
                 winPoints(currentGame, currentTurn);
                 addMatchCount(currentGame);
                 initiateNewTurn(currentGame, true);
             }
         } else {
```

Comment:
IMO pause should be in the runTurn method as well.
or, in runTurn, does the setSong method pause songs instead?